src/Components/my_profile/my_profile.js

```diff
@@ -0,0 +1,55 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import './my_profile.css';
+import { PopulateMissionProfile } from '../../redux/missions/missions';
+import { selectAllDragons } from '../../redux/dragons/dragons.redux';
+
+const Myprofile = () => {
+  const dispatch = useDispatch();
+  const missions = useSelector((state) => state.missionReducer);
+  const dragons = useSelector(selectAllDragons);
+  const rockets = useSelector((state) => state.rocket);
+
+  useEffect(() => {
+    dispatch(PopulateMissionProfile());
+  }, []);
+
+  return (
+    <div className="myProfile">
+      <section className="my-comp">
+        <h2>My Mission</h2>
+        <ul className="reservedCont">
+          { missions.map((mission) => (mission.reserved === true ? (
+            <li className="reservedItem" key={mission.mission_id}>
+              {' '}
+              {mission.mission_name}
+              {' '}
+            </li>
+          ) : null))}
```

Comment:
- [ ] **Optional:** Rather than display nothing, it's always a good user experience to have a message of what's happening. Consider adding a message like "No mission join yet!"